public_api/src/registry_types.ts

```diff
@@ -187,6 +187,12 @@ export interface MachineOnButtonPressedEventArg extends MachineCallbackArg {
  */
 export interface MachineDefinitionEvents {
   onButtonPressed?: MachineEventCallback<MachineOnButtonPressedEventArg>;
+
+  /**
+   * Called each time after a Network has finished distributing power
```

Comment:
this doc comment says the event is called when the network has finish distributing power but it is called when any storage type is distributed, not just energy.
```suggestion
   * Called after a network has completed sending machine storage allocations
```